docs/src/issues.md

```diff
@@ -0,0 +1,17 @@
+# Known issues
+
+## Missing cuda libraries
```

Comment:
I feel like `Missing CUDA libraries` is not optimal for a person that does not even know that this issue is about CUDA. What about something like
```suggestion
## `dlopen failed` is triggered by OpenMPI
```
or something similar?